assets/blocks/course-outline/frontend.js

```diff
@@ -0,0 +1,21 @@
+/* global jQuery */
+jQuery( document ).ready( () => {
+	function toggleModuleDetails() {
+		const moduleDetails = jQuery( this )
+			.closest( '.wp-block-sensei-lms-course-outline-module' )
+			.children( '.wp-block-sensei-lms-collapsible' );
+
+		moduleDetails.toggleClass( 'collapsed' );
+		jQuery( this ).toggleClass( 'dashicons-arrow-up-alt2' );
```

Comment:
We need to add `dashicons` as a dependency for the frontend stylesheet. It isn't loaded by default for most themes. For example, logged out (admin bar and query monitor both require this), this does not display on Storefront.

Reply:
We should probably use an svg icon compiled in at some point, but that's worth its own PR

Reply:
Good catch! 5e8e3b8f0e27ee37f7983e668d1ff8c55729df97

> We should probably use an svg icon compiled in at some point, but that's worth its own PR

Considered using CSS content attribute but found it a bit hacky.